lib/msf/ui/console/command_dispatcher/common.rb

```diff
@@ -98,6 +98,35 @@ def set_rhosts_from_addrs(rhosts)
     print_line
   end
 
+  def set_creds_from_database(public, private)
+    if public.empty? && private.empty?
+      print_status("invalid")
+    end
+    if active_module
+      mydatastore = active_module.datastore
+    else
+      print_error("No active module selected")
+      return
+    end
+
+    user_options = ['SMBUser', 'DBUSER', 'FTPUSER', 'HttpUsername', 'HttpProxyUser', 'BasicAuthUser', 'USERNAME']
+    pass_options = ['SMBPass', 'DBPASS', 'FTPPASS', 'HttpPassword', 'HttpProxyPass', 'BasicAuthPass', 'PASSWORD']
```

Comment:
i think the Proxy ones can be removed here, those are likely someone using their own proxy and not something they'd keep in `creds`.
I think there's a high likelihood that `Username` and `httpproxyuser` would be in the same module.

Reply:
Ok I think that can be removed as it may lead to other options be filled incorrectly

Reply:
Should basic auth ones be removed as well?